### ISSUE TITLE
[META] How to improve the module search experience

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#840]
This issue expands on https://github.com/backdrop-ops/backdropcms.org/issues/95

More prominent link to Backdrop Upgrade Status module

Provide a dedicated page for those coming from Drupal 7. Include:
- module search (indicate these are produciton ready)
- embed github repo search for `backdrop-contrib` (possible?) - indicate these are in dev
- search list of port requests (from `backdrop-ops/contrib`)
- add link to request a port (int `backdrop-ops/contrib`)

Home page:
- Place the search for modules on the home page.

Modules page:
- add text to the top of the page mentioning:
	- github contrib repo (dev modules)
	- Backdrop Upgrade Status module
- allow filtering of results pages
- allow sortting of results pages
- for the search result view mode:
	- add flag that indicates why modules are pinned
	- decrease vertical space (too much scrolling)

Top 100 Modules page:
- Link to the module search page
- Link to the contr group

Installer Module
- add text indicating that there are additional modules in development (link to contrib group)

**Comment by yorkshire-pudding** [comment]
Just wondering, is it possible with the github API that a webform could be used for
> add link to request a port (int backdrop-ops/contrib)
so that people who don't use github could request a port and have the issue created?

**Comment by jenlampton** [comment]
Yes, it's possible. But we don't have anyone working on this that's familiar enough with the GitHub API to do it.

**Comment by BWPanda** [comment]
Filtering/sorting of projects is happening in https://github.com/backdrop-ops/backdropcms.org/issues/442.
